### PR TITLE
KEE Cryptoken

### PR DESCRIPTION
**Pull request by ZeAttLabs**
Added KEE Cryptoken and image.

Updated contributing.md to refer to `contract-map.json` rather than `icon-map.json`

### CODE REVIEW

**Comment by danfinlay** [comment]
@ZeAttLabs could you link to an official source verifying this contract address?

**Comment by ZeAttLabs** [comment]
GXE is listed on Etherscan.
KEE is intended to be a somewhat cryptic token so we do not want the source to be public - but are prepared to share it with you for a visual confirmation. Would that work?

**Comment by danfinlay** [comment]
Since this list gets added to the MetaMask bundle at the moment, I'm not really ready to merge in cryptic, non-publicly-verifiable tokens yet at this time.

If you want GXE merged sooner, maybe just submit it as a separate PR.

**Comment by ZeAttLabs** [comment]
Spoke with project owner. 
Seems they are happy with letting the cat out of the bag - so source code is verified.
Do I have to submit again?

**Comment by danfinlay** [comment]
No need to resubmit, will reopen now & review later.

File: contract-map.json [hunk 1]
```diff
@@ -1,4 +1,18 @@
 {
+  "0x106b419718298f91ca576728A670597fb2e0eE4e": {
+    "name": "Genevieve exchange",
+    "logo": "gxe.png",
+    "erc20": true,
+    "symbol": "GXE",
+    "decimals": 0
```
Comment (danfinlay) [inline contract-map.json]: On the live contract, it says 10 decimals. This is not consistent. Can you clarify why this is correct?

**Comment by danfinlay** [comment]
Please comment when corrected.

**Pull request closed without merging**